samples/SqlServerCacheConcurencySample/project.json

```diff
@@ -0,0 +1,19 @@
+{
+    "commands": {
+        "install-sqlservercache": "Microsoft.Framework.Caching.SqlServer",
+        "SqlServerCacheConcurencySample": "SqlServerCacheConcurencySample"
+    },
+    "dependencies": {
+        "Microsoft.Framework.Caching.SqlServer": "1.0.0-*",
+        "Microsoft.Framework.Logging": "1.0.0-*",
```

Comment:
Are you suggesting that reference to `Console` logger alone should be enough?...I need reference to `LoggerFactory` too (to which I add the ConsoleLoggerProvider) which is part of `Logging` package.